### PR TITLE
[1LP][RFR] Service chargeback: fix SCVMM failure

### PR DESCRIPTION
**Pull request by nachandr**
{{pytest: cfme/tests/ssui/test_ssui_dashboard.py -k test_monthly_charges --long-running --use-provider complete}}

PRT results:
58 - PASS
59 - PASS

### CODE REVIEW

File: cfme/tests/ssui/test_ssui_dashboard.py [hunk 1]
```diff
@@ -85,6 +73,14 @@ def assign_chargeback_rate(new_compute_rate):
     enterprise.storageassign()
 
 
+def verify_vm_uptime(appliance, provider, vmname):
+    """Verify VM uptime is at least one hour.That is the shortest duration for
+    which VMs can be charged.
+    """
```
Comment (mkoura) [inline cfme/tests/ssui/test_ssui_dashboard.py]: REQUIRED: please use summary line followed by blank line followed by more detailed description to be consistent with our [docstrings style](https://gist.github.com/mfalesni/11694e0d563c80c3917ce6012400b93f).
I.e.
```python
def verify_vm_uptime(appliance, provider, vmname):
    """Verifies that VM uptime is at least one hour.

    One hour is the shortest duration for which VMs can be charged.
    """
```

File: cfme/tests/ssui/test_ssui_dashboard.py [hunk 2]
```diff
@@ -117,7 +113,11 @@ def verify_records_rollups_table(appliance, provider):
 
         return False
 
-    wait_for(verify_records_rollups_table, [appliance, provider], timeout=3600,
+    if provider.one_of(SCVMMProvider):
+        wait_for(verify_vm_uptime, [appliance, provider, vmname], timeout=3610,
+           message='Waiting for VM to be up for at least one hour')
+    else:
+        wait_for(verify_records_rollups_table, [appliance, provider], timeout=3600,
```
Comment (mkoura) [inline cfme/tests/ssui/test_ssui_dashboard.py]: OPTIONAL: increase delay for both ``wait_for``s so it's not run every second

File: cfme/tests/ssui/test_ssui_dashboard.py (hunk 1 above)
Comment (nachandr) [reply cfme/tests/ssui/test_ssui_dashboard.py]: done

File: cfme/tests/ssui/test_ssui_dashboard.py (hunk 2 above)
Comment (nachandr) [reply cfme/tests/ssui/test_ssui_dashboard.py]: Good idea.

File: cfme/tests/ssui/test_ssui_dashboard.py [hunk 3]
```diff
@@ -29,19 +30,6 @@
 ]
 
 
-@pytest.fixture(scope="module")
-def enable_candu(appliance):
```
Comment (mshriver) [inline cfme/tests/ssui/test_ssui_dashboard.py]: :+1:  Using the one in cfme/fixtures, awesome.

**Pull request merged**